src/flint/test/test_all.py

```diff
@@ -3678,10 +3671,8 @@ def test_matrices_eq():
         assert (A1 != A2) is False
         assert (A1 == B) is False
         assert (A1 != B) is True
-        assert (A1 == None) is False
-        assert (A1 != None) is True
-        assert (None == A1) is False
-        assert (None != A1) is True
+        assert (A1 is None) is False
+        assert (A1 is not None) is True
```

Comment:
This is also the case for some of the other changes but the code here is supposed to be testing `__eq__` so it does need to be `==` rather than `is`. In cases like this we just need a `# ruff: ignore` or whatever the flag is.